azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeErrorCode.java

```diff
@@ -44,6 +44,11 @@ public final class CallCompositeErrorCode extends ExpandableStringEnum<CallCompo
     public static final CallCompositeErrorCode NETWORK_CONNECTION_NOT_AVAILABLE =
             fromString("networkConnectionNotAvailable");
 
+    /***
+     * Dispatched when microphone is being used by other application or can not be accessed
```

Comment:
Dispatched... does not make any sense to Contoso. They have no concept of internal dispatch mechanism.

"Microphone is being used by other application or can not be accessed"

I see that this is the same issue to other error codes descriptions.
Could we change wording to those as well.